i18n/en.json

```diff
@@ -2364,7 +2364,7 @@
   },
   {
     "id": "app.import.bulk_import.unsupported_version.error",
-    "translation": ""
+    "translation": "Unsupported import data format version."
```

Comment:
Is it an unsupported version of the data format?

Reply:
the bulk imports have a first line including the version, normally version: 1. If it has a different version number, it'll fail.